PublishedModules/ProjectAlice/ContextSensitive/ContextSensitive.py

```diff
@@ -19,7 +20,7 @@ def __init__(self):
 			self._INTENT_EDIT_THIS
 		]
 
-		self._history = list()
+		self._history = deque(list(), 10)
```

Comment:
thanks for the tip, didn't know that one :)

Reply:
me neither lol. I was reading it and thought that it can not be that python has no built in queues xD